php-checks/src/main/resources/org/sonar/l10n/php/rules/php/S2068.html

```diff
@@ -1,7 +1,23 @@
-<p>Because it is easy to extract strings from a compiled application, credentials should never be hard-coded. Do so, and they're almost guaranteed to
-end up in the hands of an attacker. This is particularly true for applications that are distributed.</p>
-<p>Credentials should be stored outside of the code in a strongly-protected encrypted configuration file or database.</p>
-<h2>Noncompliant Code Example</h2>
+<p>Because it is easy to extract strings from an application source code or binary, credentials should not be hard-coded. This is particularly true
+for applications that are distributed or that are open-source.</p>
+<p>In the past, it has led to the following vulnerabilities:</p>
+<p> * <a href="http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-13466">CVE-2019-13466</a></p>
+<p> * <a href="http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-15389">CVE-2018-15389</a></p>
+<p>Credentials should be stored outside of the code in a configuration file, a database or secret management service. </p>
+<p>This rule flags instances of hard-coded credentials used in database and LDAP connections. It looks for hard-coded credentials in connection
```

Comment:
HardCodedCredentialsCheck only checks for words like `password,passwd,pwd` and it's not specific to database or LDAP. So we should remove the part of the description that does not match the implementation.

Reply:
You are right, but I don't want to duplicate the RSPEC for PHP just to omit this information. I created https://jira.sonarsource.com/browse/SONARPHP-956 to improve the rule.